### PR TITLE
Fixed template builder failures

### PR DESCRIPTION
**Pull request by schlossmj**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: robot/Cumulus/tests/browser/gift_entry/template_builder.robot [hunk 1]
```diff
@@ -40,11 +44,13 @@ Create Clone and Delete Template
     Click Gift Entry Button                   Save & Close
     Wait Until Page Contains                  This name has been used by another template. Please enter a unique name.
     Enter Value In Field                      Template Name=${new_template}
+    Click Gift Entry Button                   Next: Form Fields
+    Click Gift Entry Button                   Next: Batch Settings
```
Comment (skristem) [inline robot/Cumulus/tests/browser/gift_entry/template_builder.robot:48]: why add these 2 steps when we are not doing anything on those tabs?

File: robot/Cumulus/tests/browser/gift_entry/template_builder.robot [hunk 2]
```diff
@@ -40,11 +44,13 @@ Create Clone and Delete Template
...
     Click Gift Entry Button                   Save & Close
     Current Page Should Be                    Landing                        GE_Gift_Entry
     Click Link                                Templates
     Wait Until Page Contains                  ${new_template}
-    
+    Store Template Record Id                  ${new_template}
```
Comment (skristem) [inline robot/Cumulus/tests/browser/gift_entry/template_builder.robot:53]: This step was not originally added as the template is being deleted in the next step but makes sense if the test fails before going to the next step then it gets deleted as part of teardown

**Review by deepmalamehta — APPROVED** [review]

File: robot/Cumulus/tests/browser/gift_entry/template_builder.robot (hunk 2 above)
Comment (schlossmj) [reply robot/Cumulus/tests/browser/gift_entry/template_builder.robot:53]: Yep, that was the intention with this. I'm seeing lots of cleanup failures when I run tests, so I'm working to ensure it gets deleted no matter what.

File: robot/Cumulus/tests/browser/gift_entry/template_builder.robot (hunk 1 above)
Comment (schlossmj) [reply robot/Cumulus/tests/browser/gift_entry/template_builder.robot:48]: This was the main cause of the failure - the form throws an error if you don't click to the Form Fields tab and skip right to saving. I had it mimic creating the previous template to ensure parity.

**Review by skristem — APPROVED** [review]

**Pull request merged**